.github/workflows/add_new_versions.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         run: |
           python -u workflow_scripts/check-for-new-versions.py
         env:
-          EXCLUDED_VERSIONS: "v20.10.x,v23.0.x"
+          EXCLUDED_VERSIONS: "v20.10.x,v23.0.x,v26.1.x"
```

Comment:
Why are we adding only `v26.1.x` and not `v26.0.x`?

Reply:
Because there was a release for 26.1.5 https://github.com/moby/moby/releases/tag/v26.1.5, but there were no packages uploaded for the same at https://download.docker.com/linux/rhel/9/x86_64/stable/Packages/